java/src/main/java/com/powsybl/python/commons/Util.java

```diff
@@ -309,6 +309,8 @@ public static DataframeElementType convert(PyPowsyblApiHeader.ElementType type)
                 return DataframeElementType.BRANCH;
             case IDENTIFIABLE:
                 return DataframeElementType.IDENTIFIABLE;
+            case SUB_NETWORK:
+                return DataframeElementType.SUB_NETWORK;
```

Comment:
no need in the inverse converter ?

Reply:
Fixed

Reply:
well I think this dataframe is not updated so yes